tools/bin/ecflow/processes/main/meteorological/get_med_metfcst.py

```diff
@@ -189,7 +189,10 @@ def main():
 
         outfile = os.path.join(met_fcst_loc, '%s.nc' % (model))
         print('Conservatively remap and write to {0}'.format(outfile))
```

Comment:
Maybe add a comment that explains that this temporary file is so that we don't run into issues with the `/tmp` directory filling up (so that some clever person aka me doesn't come along later thinking they'll fix it by streamlining it but actually break it)

Reply:
done! (see below)